core/vm/contracts_suave.go

```diff
@@ -115,6 +116,20 @@ func (b *suaveRuntime) fetchDataRecords(targetBlock uint64, namespace string) ([
 	return records, nil
 }
 
+func (s *suaveRuntime) signMessage(digest []byte, signingKey string) ([]byte, error) {
```

Comment:
Why is `signingKey` not bytes as well?

Reply:
the `signingKey` in our other precompiles are also strings. Im not entirely sure why we set this precedent but I believe it was @Ruteri who did

Reply:
https://github.com/flashbots/suave-specs/blob/main/specs/rigil/precompiles.md#signethtransaction